docs/sites/development.rst

```diff
@@ -42,6 +42,7 @@ documentation and then open it in your web browser using the commands below:
 .. code-block:: bash
 
     # Run build
+    pixi run -e docs postinstall
```

Comment:
just noticed this locally: need to postinstall first because the `docs` feature is not part of the default environment.